### PR TITLE
serve static content over http(s)

### PR DESCRIPTION
**Pull request by jmazzitelli**
see https://issues.jboss.org/browse/SWS-77
and https://issues.jboss.org/browse/SWS-89

### CODE REVIEW

**Comment by jmazzitelli** [comment]
This PR now demonstrates how we can pull static js files from npm and put them in our image so they can be retrieved by a browser. Currently, I only have the openshift yaml configured for this, but to get k8s to work we just have to add a service/route to expose the http endpoint. In the current iteration, if you deploy on openshift and go to this (your hostname will be different but similar):

http://sws-default.192.168.1.15.nip.io/lib/node_modules/simplecalculatorjs/package.json

you will get the static content of that file, which was nothing more than a simple js app that was pulled down from npm during the build cycle.

The purpose of this is to demonstrate how we can integrate js content from npm into the go app. Whether this is how we really want it to work is still up for discussion.

File: config.yaml [hunk 1]
```diff
@@ -1,3 +1,7 @@
-foo:
-  string: foo
-  int: 12345
+fileserver:
+  address: localhost
+  port: 12345
```
Comment (mwringe) [inline config.yaml]: I don't think we want the fileserver to be separate from the main server hosting the API, I don't think we want a separate section for the fileserver here.

**Comment by mwringe** [comment]
I don't know if we want to include credential stuff at this point, but its good that we have it started in there.

I don't think the fileserver should not be a separate entity. I was thinking we should have something like http://myapp/console to host the console contents and something like http://myapp/api to host the api part.

**Comment by jmazzitelli** [comment]
@mwringe  that's fine. This is simply a way to get a static file server in here. We don't even have an API server yet, so there was nothing for that in here. We'll add that. This is just building things out piece by piece in short PRs to follow the JIRAs.

**Comment by jshaughn** [comment]
@jmazzitelli @mwringe I'm not sure why this is set to DoNotMerge, because the file server is not on the same port as the non-existent API server?  Maybe I'm confused but to me this moves us closer to a goal and that was iterating is about.  We can always change/massage it later.

**Comment by jmazzitelli** [comment]
This is actually working right now. The go app is serving up the sws ui skeleton from @mtho11 
Right now it is served up from the / path (so point to the OS pod's service route and you will see the skeleton react app - in my case it is http://sws-default.192.168.100.99.nip.io/

**Comment by jmazzitelli** [comment]
yes, can we please merge?

**Pull request merged**